sympy/sets/sets.py

```diff
@@ -1298,10 +1298,10 @@ def identity(self):
     def zero(self):
         return S.UniversalSet
 
-    def __new__(cls, *args, **kwargs):
-        evaluate = kwargs.get('evaluate', global_parameters.evaluate)
-
+    def __new__(cls, *args, evaluate=None):
         # flatten inputs to merge intersections and iterables
+        if evaluate==None:
+            evaluate=global_parameters.evaluate
```

Comment:
```suggestion
        if evaluate==None:
            evaluate=global_parameters.evaluate
        # flatten inputs to merge intersections and iterables
```

I'm not sure how this comment was relevant, but I'd like to preserve the way how it looked before.